config/webpacker.yml

```diff
@@ -32,5 +32,14 @@ development:
 test:
   <<: *default
 
+integration:
+  <<: *default
+
+# uncomment these to enable webpack:server
+  dev_server:
```

Comment:
are we actually using devserver here? Why?

(assets:precompile already does that)

Reply:
Because the `config/integration/development.js` fails if you don't have this in place because of a missing key error.  That said, because of the flub you noticed in https://github.com/ManageIQ/manageiq-ui-classic/pull/7194/files#r455885259 it might not be needed.

Reply:
Ah, ok, fixed in https://github.com/ManageIQ/manageiq-ui-classic/pull/7238